extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -70,6 +81,8 @@ export class AmpAdApiHandler {
    * @return {!Promise} awaiting load event for ad frame
    */
   startUp(iframe, is3p, opt_defaultVisible) {
+    const perf = installPerformanceService(window);
```

Comment:
We shouldn't be installing performance service here. Is there a way to ask it via `performanceFor`?